doc/classes/InputEventMagnifyGesture.xml

```diff
@@ -1,16 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="InputEventMagnifyGesture" inherits="InputEventGesture" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		[InputEvent] that represents a magnifying touch gesture.
+		Represents a magnify touch gesture.
```

Comment:
I think the previous version was clearer, and `a magnify touch gesture` reads weird.

Reply:
Oh, that's true. Pinch-to-zoom is what I usually hear, but I don't think we should use it.